core/src/test/scala/com/databricks/labs/remorph/transpilers/SnowflakeToDatabricksTranspilerTest.scala

```diff
@@ -10,6 +10,15 @@ class SnowflakeToDatabricksTranspilerTest extends AnyWordSpec with TranspilerTes
 
     "transpile queries" in {
 
+      /* Test BANG ! command */
+      "!set error_flag = true;" transpilesTo "--!set error_flag = true;"
+      "!define tablename=CENUSTRACKONE;" transpilesTo "--!define tablename=CENUSTRACKONE;"
+      "!print Include This Text" transpilesTo "--!print Include This Text;"
+      "!abort 77589bd1" transpilesTo "--!abort 77589bd1;"
```

Comment:
Plus the tests as above. It is possible that you may not need the EOF modification to the LEXER token.